wiki/Beatmap_Discussion/en.md

```diff
@@ -72,19 +72,21 @@ This contains the creator who submitted the beatmapset, shows the submission dat
 
 ### Watch/Unwatch
 
-This replaces beatmap subscription. You can track the beatmaps' status and open issues. This page is access via the website's Home page by clicking on `modding watchlist` located near the top of the page.
+This replaces beatmap subscription. You can track the beatmaps' status and open issues. This page is access via the website's Home page by clicking on [`modding watchlist` link](https://osu.ppy.sh/beatmapsets/watches) located near the top of the page.
 
 ### Hype Train
 
-Hype train is similar to the old star priority system. A user can submit Praise to the beatmap and it will add to the beatmap's Hype Train.
+Hype train is similar to the previous star priority system. A user can submit Praise to the beatmap and it will add to the beatmap's Hype Train.
 
 > Adding praise ❤ will raise this beatmap's hype, making it more visible for nomination and ranking!
 
 *Note: the creator can't Hype up their own beatmap. Praise must be done in the General (All Difficulties) section to add Hype. It will not work through the General or the Timeline tabs.*
```

Comment:
not really needed since the hype button was removed for the creator, this was written when it was still there for the creator.